src/pages/learn.js

```diff
@@ -240,11 +240,26 @@ const LearnPage = ({
     const { search = '', pathname = '' } = location;
     const [filterValue, setFilterValue] = useState(parseQueryString(search));
     const [textFilterQuery, setTextFilterQuery] = useState(filterValue['text']);
-    const { results: textFilterResults } = useTextFilter(textFilterQuery);
+
+    const activeContentTab = useMemo(() => {
+        const currentContentFilter = filterValue['content'];
+        if (currentContentFilter) {
+            return (
+                LearnPageTabs[filterValue['content']] || filterValue['content']
```

Comment:
```suggestion
                LearnPageTabs[currentContentFilter] || currentContentFilter
```

Should we use the variable here?